src/OrbitPaths/PathsTest.cpp

```diff
@@ -23,8 +28,24 @@ TEST(Path, AllAutoCreatedDirsExist) {
   }
 }
 
+TEST(Path, AllAutoCreatedDirsExist) {
+  auto test_fns = {CreateOrGetOrbitUserDataDir,
+                   CreateOrGetCaptureDir,
+                   CreateOrGetPresetDir,
+                   CreateOrGetOrbitAppDataDir,
+                   CreateOrGetCacheDir,
+                   CreateOrGetDumpDir,
+                   CreateOrGetLogDir};
+
+  for (auto fn : test_fns) {
+    ErrorMessageOr<std::filesystem::path> path_or_error = fn();
+    ASSERT_THAT(path_or_error, orbit_test_utils::HasValue());
+    EXPECT_TRUE(std::filesystem::is_directory(path_or_error.value()));
+  }
+}
+
 TEST(Paths, AllDirsOfFilesExist) {
```

Comment:
```suggestion
TEST(Paths, AllDirsOfFilesExistUnsafe) {
```

Reply:
Yeah I saw it once I pushed, but then another thing came up.. its fixed now

Reply:
It is not. Will fix

Reply:
done